src/main/resources/assets/minecolonies/gui/windowhutfisherman.xml

```diff
@@ -1,7 +1,7 @@
 <window xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```

Comment:
This rename to lowercase will break the window.  In code we expect windowHutFisherman.xml